ui/app/components/ui/popover/popover.component.js

```diff
@@ -4,12 +4,27 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import { I18nContext } from '../../../contexts/i18n'
 
-const Popover = ({ title, subtitle, children, footer, footerClassName, onBack, onClose }) => {
+const Popover = ({
+  title,
+  subtitle = '',
+  children,
+  footer,
+  footerClassName,
+  onBack,
+  onClose,
+  className,
+  showArrow,
```

Comment:
These two props (`className` and `showArrow`) seem to be missing from the proptypes

Reply:
addressed in 9bfe721f8